pyfr/solvers/baseadvec/kernels/transform.mako

```diff
@@ -0,0 +1,83 @@
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<% t_tol = 0.99 %>
+
+// Transforms to m = [1, 0, 0]^T
+// See Moler and Hughes 1999
+<%pyfr:macro name='transform_to' params='n, u, t, off'>
```

Comment:
I think we can avoid `off`.  With good use of () we should just be able to pass u as `'u + 1'` as an argument.

Reply:
I tried a few permutation for this but couldn't find something that worked. Ultimately the offset it required for Compressible and incompressilbe support. As well as other systems of equations. (I remember at the time when I first wrote this we tried a number of different things to tidy this up)

Reply:
Okay. We should at least allow off to be passed as an int rather than a string.  This should be relatively simple.  Better still would be to allow calling as `off=1` but that would require a tiny bit more code in the expand function.  Probably worth doing though.